package.json

```diff
@@ -4,11 +4,13 @@
   "private": true,
   "author": "HEAI-FR",
   "contributors": [
-    "Uchendu Nwachukwu"
+    "Uchendu Nwachukwu",
+    "Kim Lan Phan Hoang"
   ],
   "homepage": ".",
   "license": "MIT",
   "dependencies": {
+    "@graasp/moodle": "link:../react-app/graasp-app-moodle-analytics",
```

Comment:
This should change once the package is published on npm.

Reply:
Can you try now?